debian/rules

```diff
@@ -84,3 +84,8 @@ override_dh_auto_install:
 	# Install NSS
 	# In Rust, HOST actually refers to the build target (see README.Debian in rustc)
 	install -Dm644 target/$(DEB_HOST_RUST_TYPE)/release/libnss_authd.so debian/authd/usr/lib/$(DEB_TARGET_GNU_TYPE)/libnss_authd.so.2
+
+	# Install systemd units
+	mkdir -p debian/authd/lib/systemd/system debian/authd/usr/lib/systemd/user
```

Comment:
Why `debian/authd/usr/lib/systemd/user`?

I don't think we have user services here, do we?